dts/bindings/charger/ti,bq25180.yaml

```diff
@@ -31,3 +31,10 @@ properties:
 
   constant-charge-voltage-max-microvolt:
     default: 4200000
+
+  precharge-voltage-threshold-microvolt:
+    type: int
```

Comment:
If this value can only be 2.8V or 3.0V, then maybe it should be `type: enum` instead.

Reply:
I thought about it to, but decided against it, as all other values in https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/battery/battery.yaml are int in uv and wanted to have a somewhat consistent interface. The re-charge voltage is also an enum with either 100 or 200uv below max voltage, but this is already defined in battery.yaml. Maybe one could think about extending the battery.yaml instead of this charger, but I was unsure about that.

Reply:
Thanks for clearing this up. I am in favor of how you have implemented this change.